src/SCRIPTS/BF/PAGES/pids2.lua

```diff
@@ -48,15 +48,13 @@ if apiVersion >= 1.040 then
 end
 
 if apiVersion >= 1.040 then
-    labels[#labels + 1] = { t = "Feedforward", x = x,          y = inc.y(lineSpacing) }
-    fields[#fields + 1] = { t = "Transition",  x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 100, vals = { 9 }, scale = 100 }
-end
-
-if apiVersion >= 1.044 then
-    labels[#labels + 1] = { t = "FF Interpolate", x = x,          y = inc.y(lineSpacing) }
-    fields[#fields + 1] = { t = "Mode",           x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 4, vals = { 51 }, table = { [0] = "OFF", "ON", "AVG_2", "AVG_3", "AVG_4" } }
-    fields[#fields + 1] = { t = "Smoothness",     x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 75, vals = { 52 } }
-    fields[#fields + 1] = { t = "Boost",          x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 50, vals = { 53 } }
+    labels[#labels + 1] = { t = "Feedforward",    x = x,          y = inc.y(lineSpacing) }
+    fields[#fields + 1] = { t = "Transition",     x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 100, vals = { 9 }, scale = 100 }
+    if apiVersion >= 1.044 then
```

Comment:
lol, the conditional is not needed anymore.

Reply:
Yes it is. We want to show these parameters for api version 1.044 and up.

Reply:
Oops color blindness, reading too long I guess :man_facepalming:

Reply:
Not sure how I could misread the removed part as valid part of the code. :man_palm